src/state/slices/opportunitiesSlice/types.ts

```diff
@@ -142,7 +142,7 @@ export type GetOpportunityUserStakingDataOutput = {
 export type GetOpportunityIdsOutput = OpportunityId[]
 
 // TODO: This is not FDA-approved and should stop being consumed to make things a lot tidier without the added cholesterol
```

Comment:
We can probably remove this (wonderful) comment now, yeah?

Reply:
Unfortunately not 😢 we're still heavily relying on "earn opportunities" as a legacy of previous implementations.

The rationale was we should now be able to derive everything at component-level, from the opportunitiesSlice raw types.